tasks/test_collectd.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: "[Test] Check that collectd container is running."
+  shell: |
+    podman ps | grep {{ collectd_container_name }}
```

Comment:
Just for portability (it might not be required here?) I'd set a `__container_bin` or something variable and default it to `podman` to allow for override to `docker` if it were ever necessary.

Reply:
Although I guess your tests are specifically looking for `podman` output and might not be portable. I'm ok leaving as-is but thought I'd mention it in case you thought it made sense.

Reply:
This is used for both OSP 13 and OSP16 testing, there's a task to create a symlink so that podman -> docker if OSP 13 is used. This might be a better option.

Reply:
Ah right I do remember reading that option. Is there a way to check if you're running against an OSP13 vs 16 system? Then you could even dynamically set the appropriate container_bin.

Also you could just add a check in Ansible that looks for availability of the various bins? You'd do that via the `stat` module, then register that results and do a `set_fact` `when: result.stat.exists` or similar.

Reply:
@Leif, That should go on a separate change IMHO.

Reply:
> This is used for both OSP 13 and OSP16 testing, there's a task to create a symlink so that podman -> docker if OSP 13 is used. This might be a better option.

We are being asked to remove that symlink, I think. Selecting the correct binary is probably better than symlinking.

Reply:
@csibbitt that change is in progress, I 'll make the appropriate changes to the docs here, as requests, and then open a separate PR for removing the symlink and selecting the appropriate container_bin instead

Reply:
Yea separate change makes sense to me, just noting it, and it looks like (as Chris noted) to stop doing that :+1: 

Sounds like we're all on the same page.

Reply:
Opened an issue, and will address today